docs/UserGuide.md

```diff
@@ -102,7 +102,7 @@ Section 2 will explain the features available in easyLog.
 
 ### 2.1 Viewing help: `help`
 
-Shows available commands of the app.
+As a first-time user, you can simply type `help` to view all available commands of the app.
```

Comment:
Maybe consider adding also if you forget some of the available commands or need to check their format they can use the help function as well?